app/controllers/bonus_credits_controller.rb

```diff
@@ -3,17 +3,20 @@ class BonusCreditsController < ApplicationController
   before_action :authenticate_control!
 
   def index
-    @bonus_credits = BonusCredit.where(game: current_game).order(:round, :team_name, :amount)
+    @bonus_credits = BonusCredit.where(game: current_game).order(:round, :team_id, :amount)
   end
 
   def new
     @round = current_game.round
-    @countries = Game::COUNTRIES
     @bonus_credits = BonusCredit.new
+    @teams = Team.all
```

Comment:
Ah I see.  I misunderstood the intention of that migration.

However, I am thinking we still want to create game to teams relationship.  This is because different games might want different # of countries or country names.  For example, I know there was talk of replacing one of the player countries with a country in Africa (or maybe an African Block player nation).  While our existing setup doesn't really support that either, we probably should handle if future games want to mess with that setup.

For example, if I'm a Head of State logging in and I need to see all my incomes, the query would just be 
```ruby
# user = Head of State

# if tying team to game and making teams a collection of users
@incomes = user.team.incomes

# if teams are just generic representations of names
@incomes = Income.where(team: user.team)
```

We could also change:
```ruby
options_for_select(@teams.collect{ |team| [team.team_name, team.id]})
# to the potentially neater...
options_for_select(current_game.teams.pluck(:team_name, :team_id))
```

I think the second is more intuitive than the former.  

However, there's an additional issue I think in that you're overusing `belongs_to`.  `belongs_to` means a 1:1 relationship.  So right now Each user must have their own unique Team.  Same for Team Roles.